### DISCUSSION TITLE
Zoraxy: install pre-built binaries instead of compiling from `main`

### DISCUSSION THREAD

**Discussion by dsiebel** [tteck/Proxmox#3945] [Ideas]
I noticed that the current version of the zoraxy script [always builds from source](https://github.com/tteck/Proxmox/blob/main/install/zoraxy-install.sh#L33-L36), which can take a good while.
Since Zoraxy provides [pre-built binaries on their release page](https://github.com/tobychui/zoraxy/releases), why not install the pre-built binary from there? 
It would also make the install and update script more reproducible and keep the installed service more stable, especially when excluding unstable/candidate releases.
If you concur, I am happy to take a stab at it. I noticed that a few other install scripts I checked already use pre-built binaries from GitHub so I could draw some inspiration from there.

Just wanted to confirm this would be considered useful / an improvement, before sinking any time into it.

**Comment by tteck** [comment]
Great idea! I'll make sure to check it out as soon as I find some time.

**Comment by dsiebel** [comment]
Draft PR is here: https://github.com/tteck/Proxmox/pull/3952
I haven't tested it on Proxmox, yet. Doing that right now. When successful I'll move it to "Ready for Review".

**Comment by dsiebel** [comment]
[Second PR](https://github.com/tteck/Proxmox/pull/3963) closed as well. Bummer.
As much as I like this project, and can relate to the stress of being an OSS maintainer,
this whole experience was quite a frustrating waste of my time.
